hledger/test/journal/amounts-and-commodities.test

```diff
@@ -327,7 +327,7 @@ $ hledger -f- register -V -B cur:A "amt:<5"
 # 24. aregister report cur: and amt: query matches currency before
 # valuation/cost (note that aregister currently only tests queries other than
 # "cur:" and the account query when given the --txn-dates option. Who knows why.
-$ hledger -f- aregister a -V -B cur:A "amt:<5" --txn-dates
+$ hledger -f- aregister a -V -B cur:A "amt:<5"
```

Comment:
"aregister currently only tests queries other than cur: and the account query when given the --txn-dates option." - this can be dropped, right ? Seems wrong in all recent versions.

Reply:
Yes, I'll drop it.